receiver/gitlabreceiver/metadata.yaml

```
@@ -0,0 +1,9 @@
+type: gitlab
+
+status: 
+  class: receiver
+  stability: 
+    development: [traces]
+  distributions: [contrib]
```

If memory serves, this should be empty until we promote to alpha and actually build this receiver into the contrib distribution.

```suggestion
  distributions: []
```